operator/v1/types_network.go

```diff
@@ -459,6 +459,13 @@ type OVNKubernetesConfig struct {
 	// egressIPConfig holds the configuration for EgressIP options.
 	// +optional
 	EgressIPConfig EgressIPConfig `json:"egressIPConfig,omitempty"`
+	GatewaySubnet  string         `json:"GatewaySubnet,omitempty"`
```

Comment:
ah I see why now.. 
```suggestion
	GatewaySubnet  string         `json:"gatewaySubnet,omitempty"`
```
please pay attention to semantics, notice how other fields are being done..

Reply:
will correct